tests/storage/utils.py

```diff
@@ -448,11 +449,17 @@ def assert_num_files_in_pod(pod, expected_num_of_files):
     )
 
 
-def assert_use_populator(pvc, storage_class, cluster_csi_drivers_names):
-    expected_use_populator_value = (
-        StorageClass(name=storage_class).instance.get("provisioner") in cluster_csi_drivers_names
-    )
-    assert pvc.use_populator == expected_use_populator_value
+def assert_use_populator(pvc, storage_class, cluster_csi_drivers_names, timeout=TIMEOUT_2MIN):
```

Comment:
this function should get a ready pvc and not wait for anything; based on the description `occasionally caused NotFoundError due to the PVC not being ready`:
1. NotFoundError means the PVC does not exist (and not about it not being refady)
2. the fix here waits for `pvc.use_populator` - would it take 2 minutes to add this annotation to the PVC? isn't the PVC created with this annotation?